pkg/nsx/services/common/types.go

```diff
@@ -126,6 +126,8 @@ const (
 	DstGroupSuffix          = "dst"
 	IpSetGroupSuffix        = "ipset"
 	SharePrefix             = "share"
+	NSX_LB                  = "nsx-lb"
```

Comment:
Let's use a function to identify whether we use AVI or not, return true/false instead of types

Reply:
done